.changeset/hip-vans-itch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@finos/legend-studio": patch
+"@finos/legend-studio-manual-tests": patch
```

Comment:
Please create 1 empty changeset for `manual-tests`, this changelog message should only be in the changeset for `@finos/legend-studio`